client/my-sites/theme/main.jsx

```diff
@@ -65,9 +85,17 @@ const ThemeSheet = React.createClass( {
 		return { priceElement, themeContentElement };
 	},
 
+	getValidSections() {
+		const validSections = [];
+		validSections.push( 'overview' );
+		this.props.supportDocumentation && validSections.push( 'setup' );
+		validSections.push( 'support' );
+		return validSections;
+	},
+
 	validateSection( section ) {
-		if ( [ 'details', 'support', 'documentation' ].indexOf( section ) === -1 ) {
-			return 'details';
+		if ( this.getValidSections().indexOf( section ) === -1 ) {
+			return this.getValidSections()[0];
```

Comment:
We should actually redirect here -- otherwise the URL in the address bar will continue to read whatever invalid section was entered. (Also, server-side -- and hence indexable -- pages are going to be created for that URL!)

To that end, we'll need to provide an isomorphic wrapper covering both `page.redirect()` and its`express` counterpart, `res.redirect()`.

Clearly beyond the scope of this PR, just noting here to track it somewhere.